mock-relying-party-ui-esim/Dockerfile

```
@@ -0,0 +1,134 @@
+ENV SCOPE_USER_PROFILE=$scope_user_profile
+ENV MOCK_RP_UI_PUBLIC_URL=$mockRpUIPublicUrl
+ENV CLAIMS_USER_PROFILE=$claims_user_profile
+ENV CLAIMS_REGISTRATION=$claims_registration
+ENV DEFAULT_LANG=$default_lang
+ENV FALLBACK_LANG=$fallback_lang
+
+# Set the environment variable as a placeholder for PUBLIC_URL
+ENV PUBLIC_URL=_PUBLIC_URL_
+
+## Mock relying party portal
+WORKDIR ./app
```

Hadolint (2.14.0)

[error] Use absolute WORKDIR

(DL3000)